janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java

```diff
@@ -14,6 +14,7 @@
 
 package org.janusgraph.diskstorage.cql;
 
+
```

Comment:
Good question @farodin91. I debated doing this for `getSlice` but opted not too since slice queries should involve much less data than the full token range scans the `getKeys` methods execute. It may reduce memory pressure a bit when reading in a large adjacency list but I'm not sure it would be worth the extra level of indirection.

Reply:
We could a metric for this method, so we can decide to use this also.